<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="TerrorSplash">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>-70</y>
        <width>1100</width>
        <height>910</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>C:/Users/Matthias/Downloads/Terror.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="playButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>810</y>
        <width>340</width>
        <height>130</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="onePlayerButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>430</y>
        <width>300</width>
        <height>170</height>
       </rect>
      </property>
      <property name="text">
       <string>One Player</string>
      </property>
     </widget>
     <widget class="QPushButton" name="noPlayerButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>430</y>
        <width>300</width>
        <height>170</height>
       </rect>
      </property>
      <property name="text">
       <string>No Players</string>
      </property>
     </widget>
     <widget class="QPushButton" name="twoPlayerButton">
      <property name="geometry">
       <rect>
        <x>1290</x>
        <y>430</y>
        <width>300</width>
        <height>170</height>
       </rect>
      </property>
      <property name="text">
       <string>Two Players</string>
      </property>
     </widget>
     <widget class="QPushButton" name="simuButton">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>710</y>
        <width>300</width>
        <height>170</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU 10 Simulation Run</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Explanation">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>710</y>
        <width>300</width>
        <height>170</height>
       </rect>
      </property>
      <property name="text">
       <string>Game Explanation</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Left : L</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>80</y>
        <width>531</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>Objective : Survive and kill as many zombies as you can. Your teammate and you work together.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>240</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Right : ' (apostrophe)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>380</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Up : P</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>310</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Down : ' (semi-colon)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>450</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Shoot : , (comma)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>520</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Change gun back : N</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>590</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Change gun front : M</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>90</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Player 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>1530</x>
        <y>90</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Player 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>1490</x>
        <y>170</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Left : F</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>1490</x>
        <y>230</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Right : H</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>1490</x>
        <y>360</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Up : T</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>1490</x>
        <y>290</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Down : G</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>1490</x>
        <y>420</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Shoot : C</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>1490</x>
        <y>490</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Change gun back : Z</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>1490</x>
        <y>560</y>
        <width>111</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Change gun front : X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>270</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Green Power Up : + 50 hp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>370</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Yellow Power Up: Double Speed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>480</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Blue power up: Invincibility </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>190</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Power Ups spawn every 30 seconds and last 7.5 </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseMainMenu">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>760</y>
        <width>231</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseToGame">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>760</y>
        <width>231</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>Game Screen</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QGraphicsView" name="graphicsViewArena">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>1800</width>
        <height>800</height>
       </rect>
      </property>
      <property name="backgroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
     <widget class="QLCDNumber" name="scoreNumber">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>820</y>
        <width>440</width>
        <height>60</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pauseButton">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>920</y>
        <width>180</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Pause</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="hpBarPlayer1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>830</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="value">
       <number>23</number>
      </property>
     </widget>
     <widget class="QProgressBar" name="hpBarPlayer2">
      <property name="geometry">
       <rect>
        <x>1640</x>
        <y>830</y>
        <width>220</width>
        <height>40</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <zorder>graphicsViewArena</zorder>
     <zorder>scoreNumber</zorder>
     <zorder>hpBarPlayer1</zorder>
     <zorder>hpBarPlayer2</zorder>
     <zorder>pauseButton</zorder>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLCDNumber" name="game1lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>70</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game3lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>258</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game4lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>352</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game5lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>446</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game6lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>540</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game7lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>634</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game8lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>728</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game9lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>822</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game10lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>916</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="game2lcd">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>164</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="resultsMainMenu">
      <property name="geometry">
       <rect>
        <x>1180</x>
        <y>20</y>
        <width>721</width>
        <height>1011</height>
       </rect>
      </property>
      <property name="text">
       <string>Main Menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>519</x>
        <y>70</y>
        <width>351</width>
        <height>58</height>
       </rect>
      </property>
      <property name="text">
       <string>Chronological Order downwards. Top score is last game that was played</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>916</y>
        <width>170</width>
        <height>58</height>
       </rect>
      </property>
      <property name="text">
       <string>10th to last game that was played</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
